track/log_manager.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from django_apscheduler.models import DjangoJobExecution
+
+from track.models import *
+
+
+logger = logging.getLogger(__name__)
+
+
+class LogManager:
+
+    def start_order_log(self, order_number):
+        existing_log = OrderLog.objects.filter(order_number=order_number, is_complete=False).first()
+        if existing_log:
+            self.complete_log(existing_log)
+        order_log = OrderLog.objects.create(order_number=order_number)
+        return order_log
+
+    def start_confirmation_log(self):
+        existing_log = ConfirmationCheckLog.objects.filter(is_complete=False).first()
+        if existing_log:
+            self.complete_log(existing_log)
+        job_id = DjangoJobExecution.objects.latest('run_time').id
+        conf_log = ConfirmationCheckLog.objects.create(job_id=job_id)
+        return conf_log
+    
+    def get_confirmation_log(self):
+        return ConfirmationCheckLog.objects.filter(is_complete=False).first()
+    
+    def get_order_log(self, order_number):
+        return OrderLog.objects.filter(order_number=order_number, is_complete=False).first()
+
+    def append_to_apscheduler_log(self, log, level, message):
```

Comment:
passing the `log` object into all these methods seems unnecessary. You could just get it in this method (or by passing the order number to the order logs method).

Reply:
the functions calling this class should not even need to know there exists such as thing as an order log object (or log object).